rwgame/GameConfig.cpp

```diff
@@ -191,6 +191,7 @@ GameConfig::ParseResult GameConfig::parseConfig(GameConfig::ParseType srcType,
                 try {
                     sourceValue = srcTree.get<config_t>(key, translator);
                 } catch (pt::ptree_bad_path &e) {
+                    (void)e;
```

Comment:
What is this doing?

Reply:
It does nothing, but causes the variable to become used so the compiler doesn't warn about it being unused.

The `RW_UNUSED()` macro does this if you cannot remove the variable's name.